Add oidc to provisioning params (#779)
kyma-project/control-plane: docs/kyma-environment-broker/03-12-custom-oidc-configuration.md

---
title: Custom OIDC configuration
type: Details
---

> To create an SKR with custom OIDC (Open ID Connect) configuration you need to specify additional `oidc` provisioning parameters, for example:

Reviewer: ```suggestion
To create an SKR with a custom OIDC (Open ID Connect) configuration, specify the additional `oidc` provisioning parameters. See the example:
```
